modules/monitoring-configuring-shiftstack-remotewrite.adoc

```diff
@@ -0,0 +1,148 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/shiftstack-prometheus-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="monitoring-configuring-shiftstack-remotewrite_{context}"]
+= Remote writing to an external Prometheus instance
+
+Use remote write with both {rh-openstack-first} and {product-title} to push their metrics to an external Prometheus instance.
+
+.Prerequisites
+
+- You have access to an external Prometheus instance.
+- You have administrative access to {rh-openstack} and your cluster.
+- You have certificates for secure communication with mTLS.
+- Your Prometheus instance is configured for client TLS certificates and remote writing.
+- Telemetry is enabled in the {rh-openstack} environment.
+- The Cluster Observability Operator is installed on your cluster.
+- The monitoring stack for your cluster is configured.
+
+.Procedure
+
+. Verify that the telemetry service is running by entering the following command:
++
+[source,shell]
+----
+$ oc -n openstack get monitoringstacks metric-storage -o yaml
+----
++
+The `monitoringstacks` CRD indicates whether telemetry is operating normally.
+
+. Create a secret that is named `mtls-bundle` in the `openstack` namespace that contains HTTPS client certificates for authentication to Prometheus by entering the following command:
++
+[source,shell]
+----
+$ oc --namespace openstack \
+    create secret generic mtls-bundle \
+        --from-file=./ca.crt \
+        --from-file=osp-client.crt \
+        --from-file=osp-client.key
+----
+
+. Open the `controlplane` configuration for editing by running the following command:
++
+[source,shell]
+----
+$ oc -n openstack edit openstackcontrolplane/controlplane
+----
+
+. With the configuration open, replace the `.spec.telemetry.template.metricStorage` section so that {rh-openstack} sends metrics to Prometheus. As an example:
++
+[source,yaml]
+----
+      metricStorage:
+        customMonitoringStack:
+          alertmanagerConfig:
+            disabled: false
+          logLevel: info
+          prometheusConfig:
+            scrapeInterval: 30s
+            remoteWrite:
+            - url: https://external-prometheus.example.com/api/v1/write
```

Comment:
The user will have to replace this by the URL of their prometheus instance. Do we need to highlight it?

Reply:
Sure. I'll add a callout. At the time, just having `example.com` made sense to me, but it's better to be explicit.